frontend/javascripts/oxalis/view/action-bar/toolbar_view.js

```diff
@@ -222,6 +224,18 @@ function AdditionalSkeletonModesButtons() {
           />
         </ButtonComponent>
       </Tooltip>
+      {isMergerModeEnabled && (
+        <Tooltip title="Apply this merger mode tracing.">
+          <ButtonComponent
+            style={narrowButtonStyle}
+            onClick={() => setShowApplyMergerModeModal(true)}
+            faIcon="fa fa-cogs"
+          />
+        </Tooltip>
+      )}
+      {showApplyMergerModeModal && (
+        <ApplyMergerModeModal handleClose={() => setShowApplyMergerModeModal(false)} />
+      )}
```

Comment:
I am not totally convinced that this is the best place where to put this modal. As I did not put much thought into it there might be a better place somewhere in the code :)

Reply:
I think, it's fine. Being close to the button which opens it, creates some coherence :+1: